newsfragments/2804.misc.rst

```diff
@@ -0,0 +1 @@
+Remove an unused method of ``Amonia`` (deprecated since we do not store tmap on Ursulas anymore)
```

Comment:
`removal` newsfragment instead?

Reply:
My impression was that "removal" is for public API deprecations, but sure, I'll change to it

Reply:
I believe `Amonia` is technically a public API - although presumably would not be used by anyone other than us for testing.

Reply:
Fixed